client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { applySiteOffset } from 'lib/site/timezone';
+import { useLocalizedMoment } from 'components/localized-moment';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getSiteTimezoneValue from 'state/selectors/get-site-timezone-value';
+import getSiteGmtOffset from 'state/selectors/get-site-gmt-offset';
+import getRewindCapabilities from 'state/selectors/get-rewind-capabilities';
+import ActivityCard from 'components/activity-card';
+import ActionButtons from '../action-buttons';
+import BackupChanges from '../backup-changes';
+import useGetDisplayDate from '../use-get-display-date';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import cloudSuccessIcon from './icons/cloud-success.svg';
+
+const BackupSuccessful = ( { backup, deltas, selectedDate } ) => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const hasRealtimeBackups = useSelector( ( state ) =>
+		getRewindCapabilities( state, siteId )?.includes( 'backup-realtime' )
+	);
+
+	const moment = useLocalizedMoment();
+	const timezone = useSelector( ( state ) => getSiteTimezoneValue( state, siteId ) );
+	const gmtOffset = useSelector( ( state ) => getSiteGmtOffset( state, siteId ) );
+
+	const getDisplayDate = useGetDisplayDate();
+	const displayDate = getDisplayDate( backup.activityTs );
+	const displayDateNoLatest = getDisplayDate( backup.activityTs, false );
+
+	const today = applySiteOffset( moment(), {
+		timezone: timezone,
+		gmtOffset: gmtOffset,
+	} );
+	const isToday = selectedDate.isSame( today, 'day' );
+
+	const meta = get( backup, 'activityDescription[2].children[0]', '' );
```

Comment:
Perhaps this could be extracted in a separate function. I feel it'd make reading this component easier.

Reply:
Yeah, it isn't completely obvious what's happening here. I didn't do much refactoring of the individual statuses themselves other than incorporating selectors, but I think I can tweak this to clarify what's going on.